linkerd/app/core/src/transport/labels.rs

```diff
@@ -11,13 +12,17 @@ use std::{fmt, net::SocketAddr};
 /// Implements `FmtLabels`.
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub enum Key {
-    Accept {
-        direction: Direction,
-        tls: tls::ConditionalServerTls,
-        target_addr: SocketAddr,
-    },
-    OutboundConnect(OutboundEndpointLabels),
-    InboundConnect,
+    Server(ServerLabels),
+    OutboundClient(OutboundEndpointLabels),
+    InboundClient,
```

Comment:
alternative naming suggestion, take it or leave it: maybe instead of `Server` and `Client` these should be `Src` and `Dest`, since that's what we'll actually emit for the `peer=` label? but, maybe consistent naming with the service that generates the labels is better... :woman_shrugging:

Reply:
yeah, we're going to move away from the peer labels sooner than later